app/src/main/java/de/grobox/transportr/ui/TransportrChangeLog.kt

```diff
@@ -36,22 +36,12 @@ class TransportrChangeLog(context: Context, settingsManager: SettingsManager) :
     companion object {
 
         private fun theme(dark: Boolean): String {
-            return if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Have you checked that this is the only point where we do special things for API < 21?

Reply:
I searched for "Build.VERSION" in the entire project, and couldn't find any other place... But if you know of any, I would be happy to add it!

Reply:
No, I don't. Thanks for the confirmation!

Reply:
I checked again, and found a place that I overlooked. Just pushed a new commit to correct for it.